tests/basic_test.go

```diff
@@ -96,6 +97,87 @@ func testPolicyWithCachedCapturesAndNoDesiredStateRef(t *testing.T) {
 	})
 }
 
+func testPolicyWithFilterCaptureAndNoDesiredState(t *testing.T) {
+	t.Run("with a eqfilter capture expression and desired state that has no ref", func(t *testing.T) {
+		stateData := []byte(`
+routes:
+  running:
+  - destination: 0.0.0.0/0
+    next-hop-address: 192.168.100.1
+    next-hop-interface: eth1
+    table-id: 254
+  - destination: 1.1.1.0/24
+    next-hop-address: 192.168.100.1
+    next-hop-interface: eth1
+    table-id: 254
+  config:
+  - destination: 0.0.0.0/0
+    next-hop-address: 192.168.100.1
+    next-hop-interface: eth1
+    table-id: 254
+  - destination: 1.1.1.0/24
+    next-hop-address: 192.168.100.1
+    next-hop-interface: eth1
+    table-id: 254
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      address:
+      - ip: 10.244.0.1
+        prefix-length: 24
+      - ip: 169.254.1.0
+        prefix-length: 16
+      dhcp: false
+      enabled: true
+  - name: eth2
+    type: ethernet
+    state: down
+    ipv4:
+      address:
+      - ip: 1.2.3.4
+        prefix-length: 24
+      dhcp: false
+      enabled: false
+`)
+		const capID0 = "cap0"
+		policySpec := types.PolicySpec{
+			Capture: map[string]string{
+				capID0: `routes.running.destination=="0.0.0.0/0"`,
+			},
+			DesiredState: stateData,
+		}
+
+		beforeGenerate := time.Now()
+		sleepTime := time.Second
+		time.Sleep(sleepTime)
```

Comment:
I do not understand what are you trying to do here.
We surely are not interesting to test time... It will also most likely flake.

If you are interesting it testing the time, I guess you can have a dedicated test for it and check that for two calls we get two different time values.

Reply:
Well testing time has discover a bug, so clearly we should test the timestamp, agree with testing at different test so we increase test time only on that test.